tern/formats/json/html.py

```diff
@@ -0,0 +1,375 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017-2020 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+
+html_code = '''<!DOCTYPE html>
+   <head>
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+      <meta charset="utf-8" />
+      <title>Test page</title>
+      <script type="text/javascript" >
+         /**
+         * JSONViewer - by Roman Makudera 2016 (c) MIT licence.
```

Comment:
Where was all this code copy and pasted from?

Reply:
This code was adopted from [here](https://www.cssscript.com/json-data-tree-view/) I modified it a little bit to suit our need.